examples/seismic/skew_self_adjoint/example_iso.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+from devito import configuration
+from examples.seismic import RickerSource
+from examples.seismic.skew_self_adjoint import *
+
+configuration['language'] = 'openmp'
+configuration['log-level'] = 'DEBUG'
+
+nx, ny, nz = 401, 401, 401
```

Comment:
`nx, ny, nz` used once in this scope, unnecessary?

Reply:
done